src/components/FicheFilm/FicheFilm.css

```diff
@@ -1,25 +1,68 @@
+body {
```

Comment:
That body color broke the general background when its assembled with the main page

Reply:
ok